Hiding all before and after styles

jasonfox opened the pull request:
Found that some before and after styles were not being reset, e.g., li:before or li:after, so, I've added two directives which should hide all before and after styles.

Commit:
Hiding all before and after styles

ddelrio1986 commented:
@jasonfox why did you close this and delete your branch? Did you find that it caused some other issue?

jasonfox commented:
Just submitted a new pull-request. Rather than add display:none to *:before, *:after, I added it to the list of elements at the top so that all styles are cleared properly. I then added a content: "" directive towards the bottom for all *:before and *:after. Lastly, I realized that link pseudo classes needed resetting too, e.g., a:active, a:focus, etc. so I added those as well.
